.github/actions/pack_and_deploy/action.yml

```diff
@@ -9,6 +9,7 @@ runs:
         # on it: app-tap/gh-4983-tnt-e-assert-false-hangs.test.lua.
         echo PACKPACK_EXTRA_DOCKER_RUN_PARAMS='--init' | tee -a $GITHUB_ENV
         # create packages
+        export DEBIAN_FRONTEND=noninteractive
```

Comment:
It does not look relevant for ARM architecture. I guess I know the problem you want to solve there: we had odroid self-hosted runner, which receives postfix update (or postfix as the new dependency) on `apt-get update` below. Postfix configuration is interactive if this environment variable is not set.

So the problem is not about ARM, but about calling `apt-get update` without explicit refusal from interactive configuration. Please, move it to its own commit and give its own section with a comment (my wording is just for example):

```
# There are packages like tzdata or postfix, whose configuration
# is interactive by default. The environment variable below
# fordibs interactive configuration phase. Otherwise the CI stucks
# from time to time on `apt-get update`.
export DEBIAN_FRONTEND=noninteractive
# create packages
<...>
```

Reply:
I see that now it is moved to its own commit and has a comment. Nice.

(Please, respond to comments, when they're fixed — this way I'll receive notifications and may give feedback earlier. And it will also obvious for another reader what is a status of a particular question.)